docs/examples/02_qaoa.ipynb

```diff
@@ -174,12 +174,16 @@
     }
    ],
    "source": [
+    "USE_RUNTIME_SERVICE = False\n",
+    "\n",
+    "service = None\n",
+    "if USE_RUNTIME_SERVICE:\n",
+    "    service = QiskitRuntimeService(\n",
+    "        channel='ibm_quantum',\n",
+    "        instance='ibm-q/open/main',\n",
+    "        token='<insert your token>'\n",
```

Comment:
Sorry for the detail @akihikokuroda 😅 , I would use same terms than in previous PRs:

```
token='<Your Token>',\n",
```

That way we maintain a bit of coherence.